packages/gatsby-plugin-treat/gatsby-node.js

```diff
@@ -0,0 +1,50 @@
+const TreatPlugin = require('treat/webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+exports.onCreateBabelConfig = ({stage, actions}) => {
+  if (stage === 'develop') {
+    actions.setBabelPlugin({
+      name: `babel-plugin-treat`,
+    });
+  }
+}
+
+exports.onCreateWebpackConfig = ({stage, actions}) => {
+  let config = {};
+
+  switch (stage) {
+    case `develop`:
+      config = {
+        plugins: [
+          new TreatPlugin({}),
+        ]
+      };
+      break;
+
+    case `develop-html`:
+      break;  
+    
+    case `build-javascript`:
+      config = {
+        plugins: [
+          new TreatPlugin({
+            outputLoaders: [MiniCssExtractPlugin.loader]
+          })
+        ]
+      };
+      break;
+
+    case `build-html`:
+      config = {
+        plugins: [
+          new TreatPlugin({
+            outputCSS: false
```

Comment:
Prevent `webpack` from creating another copy of the CSS file since we already have it from the `build-javascript` step above.